syllabus/tools.md

```diff
@@ -82,6 +82,32 @@ Everything in this class will be tested with the up to date (or otherwise specif
 - Install python via [Anaconda](https://www.anaconda.com/products/individual)
 - if you use Windows, install Git with [GitBash](https://gitforwindows.org/) ([video instructions](https://youtu.be/339AEqk9c-8)).
 - if you use MacOS, install Git with the Xcode Command Line Tools. On Mavericks (10.9) or above you can do this by trying to run git from the Terminal the very first time.`git --version`
+- if you use Chrome OS, follow these instructions:
+1. Find Linux (Beta) in your settings and turn that on.
+2. Once the download finishes a Linux terminal will open, then enter the commands: sudo 
+apt-get update and sudo apt-get upgrade. These commands will ensure you are up to 
+date.
+3. Use the command sudo apt -t stretch-backports install tmux to install tmux.
+4. Next you will install nodejs, to do this, use the following commands in order, curl -sL 
+https://deb.nodesource.com/setup_14.x | sudo -E bash – then sudo apt-get install -y 
+nodejs and lastly sudo apt-get install -y build-essential.
```

Comment:
````suggestion
4. Next you will install nodejs, to do this, use the following commands:
    
    ```
    curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash 
    sudo apt-get install -y nodejs 
    sudo apt-get install -y build-essential.
    ```
````
again to fix rendering, just confirm i added the line breaks at the right point